src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -283,6 +306,128 @@ public static void ValidateEntityNamesInConfig(Dictionary<string, Entity> entity
             }
         }
 
+        /// <summary>
+        /// Helper method to validate that the rest path is correctly configured for the entity.
+        /// The rest path can only be a boolean value or a string.
+        /// If configured as a string, it should not be null/empty and should not conflict with the rest path
+        /// configured for any other entity.
+        /// </summary>
+        /// <param name="entityName">Name of the entity.</param>
+        /// <param name="restPathElement">The rest path element for the entity.</param>
+        /// <param name="restPathsForEntities">Set of unique rest paths configured for the entities in the config.</param>
+        /// <exception cref="DataApiBuilderException"></exception>
+        private static void ValidateRestPathForEntity(string entityName, JsonElement restPathElement, HashSet<string> restPathsForEntities)
+        {
+            if (restPathElement.ValueKind is JsonValueKind.Null)
```

Comment:
this is when:
```json
{
    "path": null
}
```
and not when the path property is completely left out of the config right? so if it is explicitly left out, then we would somehow resolve the path value to be null, will we?

Reply:
> and not when the path property is completely left out of the config right?

Right. No, we would not resolve to path as null if it is explicitly left out. In that case we won't even do the validations related to `path`.